doc/TestingApplications/Infrastructure/Tools/Jenkins/Jenkinsworkflow.md

```diff
@@ -1,7 +1,7 @@
 ### Jenkins Workflow
 Here's how Jenkins elements are put together and interact.
-- Developers commit the changes into repository
-- The Jenkins server checks the repository at regular intervals and pulls new changes from repository
+- Developers commit the changes into repository.
+- The Jenkins server checks the repository at regular intervals and pulls new changes from repository.
```

Comment:
I believe jenkins trigger a build as soon as an update is made on git. This line specifies Jenkins pulls changes at regular interval. could you please check into this

Reply:
Jenkins has a polling mechanism where it checks git at a given interval of time. 
The given description is correct and hence resolving the issue